app/controllers/popolo/motions_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Popolo::MotionsController < API::RestfulController
+class Popolo::MotionsController < API::MotionsController
```

Comment:
This is what makes me really excited about doing some API interoperability work: write a new controller, inherit from the Restful version of it, define a new serializer. Bam, new interoperability option, which works exactly like the one we have, in a different format.